xdsl/transforms/experimental/ConvertStencilToLLMLIR.py

```diff
@@ -32,21 +32,22 @@
 
 def GetMemRefFromField(
     input_type: FieldType[_TypeElement] | TempType[_TypeElement]
-) -> MemRefType[_TypeElement]:
+) -> memref.MemRefType[_TypeElement]:
```

Comment:
These changes feel unrelated, right? This is a Stencil refactor, AFAIU.

Reply:
True, will revert those